src/app/[assignment_sheet_id]/[tenant_id]/page.tsx

```diff
@@ -8,12 +9,69 @@ interface IAssignmentSheetPageProps {
 const AssignmentSheetPage = ({
   params: { assignment_sheet_id, tenant_id },
 }: IAssignmentSheetPageProps) => {
+  console.log(
+    `Assignment Sheet ID: ${assignment_sheet_id}, Tenant ID: ${tenant_id}`,
+  );
+
+  const rotationData = {
+    rotations: {
+      // rotation 1
+      1: {
+        startDate: '12/1',
+        endDate: '12/7',
+        categories: [
+          {
+            id: '48e5edd4-8960-4be4-8b4c-86c8c31cb8f1',
+            name: 'Cleaning',
+            tasks: [
+              {
+                id: 'd7bd18d4-60c9-4fb7-b420-3139ed8cde64',
+                title: 'Clean the kitchen',
+                description: 'Ensure the kitchen is clean and tidy.',
+                isCompleted: false,
+              },
+              {
+                id: 'a7bd18d4-60c9-4fb7-b420-3139ed8cde64',
+                title: 'Sample title',
+                description: 'Sample description.',
+                isCompleted: false,
+              },
+            ],
+          },
+        ],
+      },
```

Comment:
If isCompleted is true, checkbox is not checked on the page.
Could you please review it and make any necessary corrections?